From efa7ab581195633c685fc0ba44576df236376674 Mon Sep 17 00:00:00 2001
Message-Id: <efa7ab581195633c685fc0ba44576df236376674.1350312451.git.minovotn@redhat.com>
In-Reply-To: <aa2da19f266f7dd7345db5620ee362446fc6e806.1350312451.git.minovotn@redhat.com>
References: <aa2da19f266f7dd7345db5620ee362446fc6e806.1350312451.git.minovotn@redhat.com>
From: Alon Levy <alevy@redhat.com>
Date: Sun, 7 Oct 2012 15:31:52 +0200
Subject: [PATCH 15/27] qxl: Add missing GCC_FMT_ATTR and fix format specifier

RH-Author: Alon Levy <alevy@redhat.com>
Message-id: <1349623920-19894-9-git-send-email-alevy@redhat.com>
Patchwork-id: 42783
O-Subject: [PATCH RHEL-6.4 v2 08/16] qxl: Add missing GCC_FMT_ATTR and fix format specifier
Bugzilla: 770842
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Arnon Gilboa <agilboa@redhat.com>

From: Stefan Weil <sw@weilnetz.de>

val is an uint64_t, therefore %d was not correct.

Signed-off-by: Stefan Weil <sw@weilnetz.de>
Acked-by: Gerd Hoffmann <kraxel@redhat.com>
Signed-off-by: Stefan Hajnoczi <stefanha@linux.vnet.ibm.com>

Upstream: 5f8daf2e04709800531bc1ecbb7b49b637182777

Changes:
 val is uint32_t since ioport_write signature wasn't changed, no
 memory api, so the correct title would be "Add missing GCC_FMT_ATTR"
 (kept current title to be identical to upstream)
---
 hw/qxl.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/qxl.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/qxl.h b/hw/qxl.h
index 3f6bf47..44f1e1d 100644
--- a/hw/qxl.h
+++ b/hw/qxl.h
@@ -115,7 +115,7 @@ typedef struct PCIQXLDevice {
 
 /* qxl.c */
 void *qxl_phys2virt(PCIQXLDevice *qxl, QXLPHYSICAL phys, int group_id);
-void qxl_guest_bug(PCIQXLDevice *qxl, const char *msg, ...);
+void qxl_guest_bug(PCIQXLDevice *qxl, const char *msg, ...) GCC_FMT_ATTR(2, 3);
 
 void qxl_spice_update_area(PCIQXLDevice *qxl, uint32_t surface_id,
                            struct QXLRect *area, struct QXLRect *dirty_rects,
-- 
1.7.11.7

